client/state/desktop/middleware.js

```diff
@@ -90,6 +92,20 @@ export const desktopMiddleware = () => {
 				);
 				return next( action );
 
+			case SEND_TO_PRINTER: {
+				debug( 'Dispatching window event for action type: ', action.type );
+				const { title, contents } = action;
+				window.dispatchEvent(
+					new window.CustomEvent( NOTIFY_DESKTOP_SEND_TO_PRINTER, {
```

Comment:
We should be polyfilling `CustomEvent`, but it might be worth giving this a try in IE 11 to make sure it works. I'm unable to do so because of security restrictions on live branches, though 😕

Reply:
Thanks for the review!

My understanding is the desktop middleware is only included when Calypso is being built for the desktop app ([link](https://github.com/Automattic/wp-calypso/blob/e5b2796be1e4e11ae55e228ab5e8ccea3327fc0b/client/state/index.js#L62)). I think this means we can assume this code will be executed only in a Chrome context?

Would the polyfill for window.CustomEvent be necessary, in this case?

Reply:
Ah, no, if that's the case then there's no need for confirming whether the polyfill is working for this 👍